crates/sui-indexer/src/models_v2/objects.rs

```diff
@@ -335,10 +353,7 @@ mod tests {
         let stored_obj = StoredObject::from(indexed_obj);
 
         let sui_coin = SuiCoin::try_from(stored_obj).unwrap();
-        assert_eq!(
-            sui_coin.coin_type,
-            "0x0000000000000000000000000000000000000000000000000000000000000002::sui::SUI"
-        );
+        assert_eq!(sui_coin.coin_type, "0x2::sui::SUI");
```

Comment:
LG, can you add a test for `Balance` to make sure it also uses the short format?

Reply:
Good point. Will do